force-app/main/default/lwc/serviceDeliveryRow/serviceDeliveryRow.js

```diff
@@ -398,7 +520,7 @@ export default class ServiceDeliveryRow extends LightningElement {
                 }
             });
 
-            if (hasContact) {
+            if (this.hasContact) {
```

Comment:
This should only run if the default values passed in, include a contactId. Should not just be if the localFieldSet includes the field.

Reply:
Good catch. I will add that check